tf-modules/report-executions/package.json

```diff
@@ -8,9 +8,8 @@
     "node": ">=8.10.0"
   },
   "scripts": {
-    "test": "ava",
-    "build": "rm -rf dist && mkdir dist && webpack",
-    "watch": "rm -rf dist && mkdir dist && webpack --progress -w",
+    "build": "rm -rf dist && mkdir dist",
```

Comment:
When I moved the Lambda code out of the TF modules and into the API package, I had to disable the build/watch scripts or else the CI build would fail trying to build these packages